content/news/2022-04-12-bevy-0.7/index.md

```diff
@@ -0,0 +1,383 @@
++++
+title = "Bevy 0.7"
+date = 2022-04-12
+[extra]
+author = "Carter Anderson"
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
+image = "FIXME.png"
+show_image = true
+image_subtitle = ""
+image_subtitle_link = ""
++++
+
+Thanks to **X** contributors, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.7** release on [crates.io](https://crates.io/crates/bevy)!
+
+For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out [Quick Start Guide](/learn/book/getting-started/) to get started. Bevy is also free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
+
+To update an existing Bevy App or Plugin to **Bevy 0.7**, check out our [0.6 to 0.7 Migration Guide](/learn/book/migration-guides/0.6-0.7/).
+
+As always, there are a _ton_ of new features, bug fixes, and quality of life tweaks in this release, but here are some of the highlights: 
+
+* Skeletal animation and mesh skinning
+* GLTF animation importing
+* Unlimited* point lights in a scene
+* Improved clustered forward rendering: dynamic/adaptive clustering, faster and more accurate cluster assignment
+* Compressed texture support (KTX2 / DDS / .basis): load more textures in a scene, faster
+* Compute shader / pipeline specialization: Bevy's flexible shader system was ported to compute shaders, enabling hot reloading, shader defs, and shader imports
+* Render to texture: cameras can now be configured to render to a texture instead of a window
+* Flexible user-customizable mesh vertex layouts in shaders
+* ECS improvements: Order systems using their names, Query::get_many, use conflicting parameters in systems via ParamSets, WorldQuery derives
+* More audio control: pause, volume, speed, and looping
+* Hot reloading for "plugin-provided / built-in assets"
```

Comment:
I think it's worth calling out that shader hot-reloading now works.

Reply:
Yeah that was the motivating use case / worth calling out explicitly. We will also touch on that in the "in-shader import path" coverage, as those features are intertwined.

Reply:
I've actually removed this section because it has become clear that this feature isn't ready for user consumption yet / only really works for internal bevy shaders atm.